R/Filter.R

```diff
@@ -75,7 +75,7 @@ listFilterMethods = function(desc = TRUE, tasks = FALSE, features = FALSE) {
     package = vcapply(extractSubList(filters, "pkg"), collapse)
   )
   if (desc)
-    df$desc = extractSubList(filters, "desc")
+    df$desc = clipString(extractSubList(filters, "desc"), len = 40)
```

Comment:
Are you sure you want to clip the string here? I think @berndbischl pointed to a separate S3 class for the listFilterMethods result and clip the desc just in the printer